### PR TITLE
[YUNIKORN-2443] Update the features page with the event system

### PR DESCRIPTION
**Pull request by pbacsko**
### What is this PR for?
Add a short description about the event system to the core features page. 

### What type of PR is it?
* [ ] - Bug Fix
* [x] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-2443

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

File: docs/get_started/core_features.md [hunk 1]
```diff
@@ -93,3 +93,7 @@ The [Partition and Queue Configuration](user_guide/queue_config.md) provides con
 ## CPU Architecture support
 YuniKorn supports running on ARM as well as on AMD/Intel CPUs.
 With the release of YuniKorn 1.1.0, prebuilt convenience images for both architectures are provided in docker hub.
+
+## Event system and application history tracking
+Whenever something relevant happens inside the scheduler (eg. request is added, allocation happens in a queue, node removal, queue or user quota is exceeded, etc) an appropriate event is generated. This enables users to reliably generate usage statistics about their workloads over time. Since the event structure is well-defined, this is much more suitable for automated processing than logs. Application history (state transitions & allocations), node, queue, user and group resource usage can be examined without having to look at the logs.
```
Comment (chia7712) [inline docs/get_started/core_features.md:98]: Maybe we should have it be multiple-lines.

File: docs/get_started/core_features.md [hunk 2]
```diff
@@ -93,3 +93,7 @@ The [Partition and Queue Configuration](user_guide/queue_config.md) provides con
...
+The events accessible on the REST interface in two ways: batch or streaming. The batch query simply returns the list of events, while the streaming endpoint keeps the connection open and new events are immediately sent to the client.
```
Comment (chia7712) [inline docs/get_started/core_features.md:99]: How about adding the APIs link here?

**Review by chia7712 — APPROVED** [review]
LGTM. two trivial comments below.

File: docs/get_started/core_features.md [hunk 3]
```diff
@@ -95,5 +95,7 @@ YuniKorn supports running on ARM as well as on AMD/Intel CPUs.
...
-The events accessible on the REST interface in two ways: batch or streaming. The batch query simply returns the list of events, while the streaming endpoint keeps the connection open and new events are immediately sent to the client.
+Whenever something relevant happens inside the scheduler (eg. request is added, allocation happens in a queue, node removal, queue or user quota is exceeded, etc) an appropriate event is generated.
+This enables users to reliably generate usage statistics about their workloads over time. Since the event structure is well-defined, this is much more suitable for automated processing than logs.
+Application history (state transitions & allocations), node, queue, user and group resource usage can be examined without having to look at the logs.
+The events accessible on the REST interface in two ways: batch or streaming. A [batch query](api/scheduler.md#batch-events) simply returns the list of events, while the [streaming](api/scheduler.md#event-stream) endpoint keeps the connection open and new events are immediately sent to the client.
```
Comment (chia7712) [inline docs/get_started/core_features.md:101]: the path should be `../api/scheduler.md#batch-events` and `../api/scheduler.md#event-stream`

**Review by chia7712 — APPROVED** [review]

**Review by craigcondit — APPROVED** [review]
+1 LGTM.

**Pull request closed without merging**